XcodeServerSDKTests/RepositoryTests.swift

```diff
@@ -79,5 +79,35 @@ class RepositoryTests: XCTestCase {
         sshEnum = .LoggedInReadWrite
         XCTAssertEqual(sshEnum.toString(), "All logged in users can read and write")
     }
+    
+    // MARK: API Routes tests
+    func testGetRepositories() {
+        let expectation = self.expectationWithDescription("Get Repositories")
+        let server = self.getRecordingXcodeServer("get_repositories")
+        
+        server.getRepositories() { (repositories, error) in
+            XCTAssertNil(error, "Error should be nil")
+            XCTAssertNotNil(repositories, "Repositories shouldn't be nil")
+            
+            if let repos = repositories {
+                XCTAssertEqual(repos.count, 2, "There should be two repositories available")
+                
+                for (index, repo) in repos.enumerate() {
+                    XCTAssertEqual(repo.name, "Test\(index + 1)")
```

Comment:
Should we be making the assumption that the order of returned repositories will always be the same? I'm not sure how much is guaranteed in the API. Maybe instead just create a set of names and make sure all are present instead of asserting that they will be there in this exact order.